lstchain/tests/test_lstchain.py

```diff
@@ -21,6 +22,7 @@
 file_model_energy = os.path.join(test_dir, 'reg_energy.sav')
 file_model_disp = os.path.join(test_dir, 'reg_disp_vector.sav')
 file_model_gh_sep = os.path.join(test_dir, 'cls_gh.sav')
+r0_file = Path('test_data/real/R0/20200218/LST-1.1.Run02008.0000_first50.fits.fz')
```

Comment:
With this filename, r0_to_dl1 is complaining:
```
Filename dl1_LST-1.1.Run02008.0000_first50.h5 does not match pattern re.compile('dl1_LST-(\\d+)(?:.(\\d+))?.Run(\\d+).(\\d+)(?:.fits)?.(?:h5|hdf5|hdf)')
```
Shall we modify it or just copy it as `test_data/real/R0/20200218/LST-1.1.Run02008.0000.fits.fz`?

Reply:
I would just update the regex to also allow additional suffixes

Reply:
I agree. The PR looks good to me then.